www/modules/centreon-open-tickets/providers/Abstract/templates/clone.ihtml

```diff
@@ -12,7 +12,7 @@
 
 <div id="{$cloneId}_controls">
     <div id="{$cloneId}_add">
-        <p style="cursor:pointer; color: #4ccaff;">{t}+ Add a new entry{/t}</p>
+        <p class="add-new-entry">{t}+ Add a new entry{/t}</p>
```

Comment:
pointer has disappeared

Reply:
we replace inline style with class add-new-entry .add-new-entry {
    cursor: pointer;
    color: var(--add-new-entry-font-color);
}